index.html

```diff
@@ -250,5 +250,8 @@ <h1>About <br />Myself</h1>
        
       </ul>
```

Comment:
Nice work though, but i think it is better not to leave any blank lines in your code where unnecessary. Doing this your code looks your code unclear.
Kindly please remove that blank line. Thank you

Reply:
Resolved this.